### DISCUSSION TITLE
Im lost 😅

### DISCUSSION THREAD

**Discussion by magoo94** [Jyers/Marlin#155] [Q&A] [answered]
This is like my 3rd time on github I hardly understand how this works on here could anyone point me in the direction of the firmware version I'd install I'm already using marlin 2.0.x with a BLTOUCH and my raspberryPI kit will be here tomorrow so ocoto print will be running soon on this aswell pretty soon if that matters last firmware I found was a  link straight to the file download with bltouch ready to use is there a precompiled version with the BLT touch ready along with octo print for a 4.2.2 board?

**Comment by darco415** [comment, marked as answer]
https://github.com/Jyers/Marlin/releases

**Reply by magoo94** [reply]
Thank you for the link I tried it out last night everything worked besides my BLt going outside of the build area for probing. I have one more question if you dont mind answering like I said this is literally the first time I ever did anything close to this and I'm very lost I changed the bltouches postioning when it levels to centering it on the bed with a 5x5 version then I go and build the firmware or whatever and its fails all I did is change for numbers is there a easier way todo this or a video besides g
Teaching techs guide I'm "LOST"once again lol

**Reply by Jyers** [reply]
Unfortunately there is not easier way to compile. However, if all you need to do is change the xy probe offsets, you can do that via the on screen menu, so you can simply use one of the precompiled versions instead of compiling your own.
